### DISCUSSION TITLE
If I were to create a mod, how would I upload it to GitHub?

### DISCUSSION THREAD

**Discussion by Imstupiditstrue** [R74nCom/sandboxels#899] [Q&A]
Im still trying to learn how to make a mod but after that's done what do I do?

**Comment by Bufka2011** [comment]
You should create a fork of this repository and add your mod there and then open pull request.

**Reply by Imstupiditstrue** [reply]
How do I do that (I never use GitHub really)

**Reply by pennylicker6** [reply]
you still waiting?

**Reply by Imstupiditstrue** [reply]
> you still waiting?

Yep

**Comment by P0gDog** [comment]
Fork the resp for sandboxels, add your mod to the mods folder, code it, then send a push request to the main sandboxels respritory.
